docusaurus.config.js

```diff
@@ -64,12 +64,19 @@ const config = {
         contextualSearch: true,
         searchPagePath: '/search',
       },
-      "announcementBar": {
-        "id": "calico_ebpf",
-        "content": "Use Calico <img src=\"/img/brands/ebpf_logo.svg\" style=\"height:1.5rem; margin: 0 5px 0 5px; display: inline-flex;\" alt=\"eBPF\" /> dataplane to enhance your Kubernetes networking performance. Click <a href=\"https://docs.tigera.io/calico/latest/operations/ebpf/use-cases-ebpf/\">here </a> to learn more.",
-        "backgroundColor": "#FCE181",
-        "textColor": "#000",
-        "isCloseable": true
+      //"announcementBar": {
+        //"id": "calico_ebpf",
+        //"content": "Use Calico <img src=\"/img/brands/ebpf_logo.svg\" style=\"height:1.5rem; margin: 0 5px 0 5px; display: inline-flex;\" alt=\"eBPF\" /> dataplane to enhance your Kubernetes networking performance. Click <a href=\"https://docs.tigera.io/calico/latest/operations/ebpf/use-cases-ebpf/\">here </a> to learn more.",
+        //"backgroundColor": "#FCE181",
+        //"textColor": "#000",
+        //"isCloseable":false
+
+        "announcementBar": {
+          "id": "calico_slack",
+          "content": "Get your answers from Calico Open Source creators and users on our community slack channel. Join <a href=\"https://slack.projectcalico.org/\">here </a>.",
```

Comment:
```suggestion
          "content": "Got questions? <a href=\"https://slack.projectcalico.org/\">Join our community Slack channel</> to get help from Calico Open Source creators and other community members.",
```
I'd definitely like to move away from the "click here" phrasing. The rest is just a suggestion.